packages/govuk-frontend/src/govuk/all.mjs

```diff
@@ -29,76 +29,39 @@ function initAll(config) {
     return
   }
 
+  const components = /** @type {const} */ ([
+    [Accordion, config.accordion],
+    [Button, config.button],
+    [CharacterCount, config.characterCount],
+    [Checkboxes],
+    [ErrorSummary, config.errorSummary],
+    [ExitThisPage, config.exitThisPage],
+    [Header],
+    [NotificationBanner, config.notificationBanner],
+    [Radios],
+    [SkipLink],
+    [Tabs]
+  ])
+
   // Allow the user to initialise GOV.UK Frontend in only certain sections of the page
   // Defaults to the entire document if nothing is set.
   const $scope = config.scope instanceof HTMLElement ? config.scope : document
 
-  const $accordions = $scope.querySelectorAll('[data-module="govuk-accordion"]')
-  $accordions.forEach(($accordion) => {
-    new Accordion($accordion, config.accordion)
-  })
-
-  const $buttons = $scope.querySelectorAll('[data-module="govuk-button"]')
-  $buttons.forEach(($button) => {
-    new Button($button, config.button)
-  })
-
-  const $characterCounts = $scope.querySelectorAll(
-    '[data-module="govuk-character-count"]'
-  )
-  $characterCounts.forEach(($characterCount) => {
-    new CharacterCount($characterCount, config.characterCount)
-  })
-
-  const $checkboxes = $scope.querySelectorAll(
-    '[data-module="govuk-checkboxes"]'
-  )
-  $checkboxes.forEach(($checkbox) => {
-    new Checkboxes($checkbox)
-  })
-
-  // Find first error summary module to enhance.
-  const $errorSummary = $scope.querySelector(
-    '[data-module="govuk-error-summary"]'
-  )
-  if ($errorSummary) {
-    new ErrorSummary($errorSummary, config.errorSummary)
-  }
-
-  const $exitThisPageButtons = $scope.querySelectorAll(
-    '[data-module="govuk-exit-this-page"]'
-  )
-  $exitThisPageButtons.forEach(($button) => {
-    new ExitThisPage($button, config.exitThisPage)
-  })
-
-  // Find first header module to enhance.
-  const $header = $scope.querySelector('[data-module="govuk-header"]')
-  if ($header) {
-    new Header($header)
-  }
-
-  const $notificationBanners = $scope.querySelectorAll(
-    '[data-module="govuk-notification-banner"]'
-  )
-  $notificationBanners.forEach(($notificationBanner) => {
-    new NotificationBanner($notificationBanner, config.notificationBanner)
-  })
-
-  const $radios = $scope.querySelectorAll('[data-module="govuk-radios"]')
-  $radios.forEach(($radio) => {
-    new Radios($radio)
-  })
-
-  // Find first skip link module to enhance.
-  const $skipLink = $scope.querySelector('[data-module="govuk-skip-link"]')
-  if ($skipLink) {
-    new SkipLink($skipLink)
-  }
-
-  const $tabs = $scope.querySelectorAll('[data-module="govuk-tabs"]')
-  $tabs.forEach(($tabs) => {
-    new Tabs($tabs)
+  components.forEach(([Component, config]) => {
```

Comment:
Need to double check browser support for this line

Reply:
Isn't that the kind of things that Babel abstracts for us? It seems a transform that's pretty straightforward for it anyway.

Reply:
Sometimes a polyfill is needed, but we haven't enabled them. Just need to double check